javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,20 @@
+const dogYearOfBirth = prompt("Enter the birthyear of the dog : ");
+const dogYearFuture = prompt("Enter the future year you want to be for the dog ?");
+let dogYear = dogYearFuture - dogYearOfBirth;
+//let shouldShowResultInDogYears = false;
+let shouldShowResultInDogYears = confirm("Shoud result show in dog years ? hint: if yes?click Ok else click calcel:");
```

Comment:
These should be `const`s as they don't change.

Reply:
If you want to make it more readable consider adding the `newline` character `\n` to your confirm box.

It looks like this:
```javascript
const shouldShowResultInDogYears = confirm("Should result show in dog years? \nHint: if 'Yes' click 'Ok' else click 'Cancel'.");
```